packages/orchestrator-ui-components/src/rtk/endpoints/inSync.ts

```diff
@@ -0,0 +1,22 @@
+import { Subscription } from '@/types';
+
+import { BaseQueryTypes, orchestratorApi } from '../api';
+
+const inSyncApi = orchestratorApi.injectEndpoints({
+    endpoints: (build) => ({
+        setSubscriptionInSync: build.mutation<
+            void,
+            Subscription['subscriptionId']
+        >({
+            query: (subscriptionId) => ({
+                url: `subscriptions/${subscriptionId}/set_in_sync`,
+                method: 'PUT',
+            }),
+            extraOptions: {
+                baseQueryType: BaseQueryTypes.fetch,
+            },
```

Comment:
Assuming this action should invalidate a cache-key somewhere

Reply:
No. This makes the query use fetch instead of graphql

Reply:
The placement of my comment is not ideal, but the cache key invalidation should happen somewhere in this object.

Reply:
There is no caching logic needed here. I'm doing an optimistic update on the subscriptionDetail

Reply:
lets do this later when all calls are in RTK